.github/workflows/deploy.yml

```diff
@@ -1,5 +1,11 @@
 name: Deploy multinet
-on: [repository_dispatch, push]
+on:
+  push:
+    branches:
+      - master
+  repository_dispatch:
+    branches:
+      - master
```

Comment:
Not sure this part is right. I don't think `repository_dispatch` takes a `branches` option. You can filter by event type, but I think we should just leave it without any constraints right now (since that's how it was before). Or, since I'm not sure how to just include a "blank" `repository_dispatch` section, you could have it filter on whatever dispatch type we're using (I forget what the magic word is).

Reply:
So I looked at the raw event dump and it seems like it does the dispatch to branch master so this was my solution. I had tried to leave it blank initially but that didn't work

Reply:
But repository dispatch events aren't associated with a branch, are they? Don't they just... arrive at the target repository and get picked up unconditionally by a filter like this one?

I think a better idea here is to filter on the dispatch type, as in this example: https://github.com/marketplace/actions/repository-dispatch

Reply:
Okay, I followed their example and did:
```
repository_dispatch:
    types: [build_application]
```